lib/wireprotocol/2_6_support.js

```diff
@@ -104,12 +104,13 @@ WireProtocol.prototype.killCursor = function(bson, ns, cursorState, pool, callba
     try {
       pool.write(killCursor, options, callback);
     } catch (err) {
-      callback(err, null);
+      if (typeof callback === 'function') {
+        callback(err, null);
+      } else {
+        console.warn(err);
```

Comment:
should we be `console.warn`ing here? I assume any low-level wire protocol errors, should be logged to a `Logger` object.

Reply:
Generally: yes. Here's my argument for this one: this is explicitly the case where you call killCursors (a legacy command people are not likely using anymore) without a callback (which we expressly recommend against). Point being, I'd like to fix this if and when someone actually complains about it, because we don't easily have access to a defined logger here and it's a very niche situation.